packages/shared/src/components/squads/moderation/SquadModerationItem.tsx

```diff
@@ -99,7 +99,9 @@ export function SquadModerationItem(
           </Typography>
           <PostTags className="!mx-0 min-w-full" tags={post?.tags} />
         </div>
-        <CardImage src={image || post?.image} />
+        <div className="flex-1">
+          <CardImage className="mx-auto" src={image || post?.image} />
```

Comment:
Wouldn't it just be enough to add flex-1 here?
Or flex-col on the parent so they split evenly?

Reply:
> Wouldn't it just be enough to add flex-1 here?
No, that didn't work, I tried. 😅

Also, the parent already is flex col for mobile, but on desktop they're supposed to be side-by-side, not stacked.
I did try it just now in case this was some chad css hack that had flown over my head, but alas, it was not 😂